lib/theme_check/liquid_node.rb

```diff
@@ -166,6 +171,10 @@ def block?
       block_tag? || block_body? || document?
     end
 
+    def schema?
+      @value.is_a?(ThemeCheck::Tags::Schema)
```

Comment:
I think that won't work when ThemeCheck is running in Core or SFR

Reply:
Why not? It's our namespace? Didn't we copy the code over?